SECURITY.md

```diff
@@ -0,0 +1,19 @@
+# Security Policy
+
+## Supported Versions
```

Comment:
What are the odds that this actually gets updated regularly like it should?  Suppose it could be part of @wallentx 's process.....

Reply:
we should add automation to update this. IMO